src/starcounter-debug-aid.html

```diff
@@ -173,6 +81,11 @@
                 //this.$.errorDialog.reset();
             },
             openDialog: function () {
+                if (!Polymer.isInstance(this.$.ui)) {
+                    Polymer.Base.importHref("src/starcounter-debug-aid-ui.html", () => {
```

Comment:
Path `src/starcounter-debug-aid-ui.html` fails to work in a Starcounter app. Polymer resolves it to an absolute link (`/src/starcounter-debug-aid-ui.html`). 

It needs to be relative to `starcounter-debug-aid.html` or it won't load in a Starcounter app, which has the external dependencies usually under `/sys/`.

![2016-12-09 15_41_08-launcher](https://cloud.githubusercontent.com/assets/566463/21052561/f204d54e-be25-11e6-9dfc-00a11ce4c73a.png)